frontend/javascripts/oxalis/view/components/setting_input_views.tsx

```diff
@@ -477,33 +493,87 @@ export class UserBoundingBoxInput extends React.PureComponent<UserBoundingBoxInp
       isVisible,
       onDelete,
       onExport,
+      onRegisterSegmentsForBB,
       isExportEnabled,
       onGoToBoundingBox,
       disabled,
       isLockedByOwner,
       isOwner,
     } = this.props;
     const upscaledColor = color.map((colorPart) => colorPart * 255) as any as Vector3;
-    const iconStyle = {
-      marginRight: 0,
+    const marginRightStyle = {
+      marginRight: 6,
+    };
+    const marginLeftStyle = {
       marginLeft: 6,
     };
-    const disabledIconStyle = { ...iconStyle, opacity: 0.5, cursor: "not-allowed" };
-    const exportIconStyle = isExportEnabled ? iconStyle : disabledIconStyle;
-    const exportButtonTooltip = isExportEnabled
-      ? "Export data from this bounding box."
-      : messages["data.bounding_box_export_not_supported"];
-    const exportColumn = isExportEnabled ? (
-      <Col span={2}>
-        <Tooltip title={exportButtonTooltip} placement="topRight">
-          <DownloadOutlined onClick={onExport} style={exportIconStyle} />
-        </Tooltip>
-      </Col>
-    ) : null;
+    const disabledIconStyle = { ...marginRightStyle, opacity: 0.5, cursor: "not-allowed" };
+    const exportIconStyle = isExportEnabled ? marginRightStyle : disabledIconStyle;
+    const exportButton = (
+      <>
+        <DownloadOutlined style={exportIconStyle} /> Export data
+      </>
+    );
     const editingDisallowedExplanation = messages["tracing.read_only_mode_notification"](
       isLockedByOwner,
       isOwner,
     );
+
+    const getContextMenu = () => {
+      const items: MenuProps["items"] = [
+        {
+          key: "registerSegments",
+          label: (
+            <>
+              <ScanOutlined style={marginRightStyle} /> {""}
```

Comment:
I think you can put the icon into an `icon` property (next to key and label). same for the others